src/main/java/input/file/DataFile.java

```diff
@@ -0,0 +1,116 @@
+package input.file;
+
+import java.io.*;
+
+import exceptions.DataFileNotRecognizedException;
+/**
+ * Class for a datafile.
+ *
+ * @author Paul
+ */
+public abstract class DataFile {
+
+	protected String path;
+	protected int startLine;
+	protected int endLine;
```

Comment:
Je schrijft in geen van de subclasses van DataFile naar deze variabelen, dus ze mogen private zijn.